python/tank/commands/util.py

```diff
@@ -33,3 +33,19 @@ def should_use_legacy_yaml_parser(args):
         legacy_parser = False
 
     return (legacy_parser, args)
+
+
+def is_git_repo_uri(uri: str) -> bool:
+    """
+    Checks if the given config URI is a Git repository URI.
+    """
+    return any(
+        (
+            # GitHub repo URLs end with .git
+            uri.endswith(".git"),
+            # Git SSH URLs start with git@
+            uri.startswith("git@"),
+            # Azure DevOps repo URLs contain `_git` in the HTTPS URLs
+            uri.startswith("https://") and "_git" in uri,
```

Comment:
Wow this is not the best. I guess it's OK for a quick fix but we'll need another strategy later. Why do we need to identify this anyway?